modules/hal_nordic/nrfx/Kconfig

```diff
@@ -129,7 +129,17 @@ config NRFX_PWM3
 
 config NRFX_QDEC
 	bool "Enable QDEC driver"
-	depends on HAS_HW_NRF_QDEC
+	depends on HAS_HW_NRF_QDEC || HAS_HW_NRF_QDEC0 || HAS_HW_NRF_QDEC1
```

Comment:
```suggestion
	depends on HAS_HW_NRF_QDEC || HAS_HW_NRF_QDEC0
```

Reply:
OK, so you're accepting to keep the HAS_HW_NRF_QDEC0 as "used" symbol here?

Reply:
yes